js/dragquestion.js

```diff
@@ -868,6 +900,9 @@ H5P.DragQuestion = (function ($) {
     self.y = element.y;
     self.width = element.width;
     self.height = element.height;
+    if (self.height < 10) {
```

Comment:
Per an experiment that I ran, the calculation of self.height turns out to be the same even if the units are set to em's or percentages.